utils/checkCookie.js

```diff
@@ -0,0 +1,10 @@
+export const checkCookie = (cookie) => {
+  const cookieArr = cookie?.split("; ");
+  const accessTokenFromCookie = cookieArr
+    ?.find((item) => item.split("=")[0] === "weather_access")
+    ?.split("=")[1];
+  const refreshTokenFromCookie = cookieArr
+    ?.find((item) => item.split("=")[0] === "weather_refresh")
+    ?.split("=")[1];
+  return [accessTokenFromCookie, refreshTokenFromCookie];
+};
```

Comment:
Also it is better to work with cookies using some express middleware, like cookie-parser

Reply:
Thanks for the comment, I took note, will study this middleware